src/api/app/models/package.rb

```diff
@@ -1386,25 +1386,28 @@ def api_obj
     self
   end
 
+  #### WARNING: these operations run in build object, not this package object
   def rebuild(params)
-    backend_build_command(:rebuild, params.slice(:package, :arch, :repository))
+    backend_build_command(:rebuild, params[:project], params.slice(:package, :arch, :repository))
```

Comment:
Because then it ends up in parameter list of the backend command. So we also would have to filter this.
See https://github.com/openSUSE/open-build-service/pull/3269/files#diff-37dd923f1e6421bf4aa8375fbe11ad01R1409